internal/server/openapi.go

```diff
@@ -24,6 +24,7 @@ var (
 	pathIDReplacer            = regexp.MustCompile(`:\w+`)
 	funcPartialNameToTagNames = map[string]string{
 		"Grant":       "Grants",
+		"Identities":  "Identities",
```

Comment:
Is this necessarily? From the context, it seems to be setting up aliases so `Identities->Identities` is redundant

Reply:
This one is for categorizing names, I missed it the first time. Without it this happens:
![image](https://user-images.githubusercontent.com/5853428/163182320-084b7bf8-9b20-49ce-9828-5e582ea892eb.png)